### PR TITLE
fix: select wallet page is empty

### LINKED ISSUE #4878: [Bug]Connect Mask wallet didn't go to create/import wallet page in popup
### How to reproduce?

1. Change wallet from change wallet page;
2. Since no any mask wallet in popup, connect mask wallet;
3. Check the popup.

### What happened?
Connect Mask wallet from change wallet page, didn't go to create/import wallet page in popup.
Screen Shot 2021-11-22 at 2 36 50 PM

### What should happen?
Since no any mask wallet in popup, connect mask wallet should go to create/import wallet page in popup.

### Mask Version:

- [ ] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [x] master branch
- [ ] Build of version/commit hash:

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [x] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [x] Latest Chrome (stable channel)
- [ ] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

### PR DESCRIPTION
**Pull request by nuanyang233**
closes #4878

### CODE REVIEW

**Review by Jack-Works — APPROVED** [review]

**Comment by wenluomask** [comment]
Checked.

File: packages/mask/src/plugins/Wallet/services/account.ts [hunk 1]
```diff
@@ -75,9 +75,9 @@ export async function updateMaskAccount(options: { account?: string; chainId?: C
     if (!options.chainId && options.networkType) options.chainId = getChainIdFromNetworkType(options.networkType)
 
     const { account, chainId, networkType } = options
-
     if (chainId) currentMaskWalletChainIdSettings.value = chainId
     if (networkType) currentMaskWalletNetworkSettings.value = networkType
+    if (!account) currentMaskWalletAccountSettings.value = ''
     if (account && EthereumAddress.isValid(account)) currentMaskWalletAccountSettings.value = account
```
Comment (zhouhanseng) [inline packages/mask/src/plugins/Wallet/services/account.ts:81]:
```suggestion
    if (!account || !EthereumAddress.isValid(account)) currentMaskWalletAccountSettings.value = ''
    currentMaskWalletAccountSettings.value = account

```

File: packages/mask/src/extension/popups/pages/Wallet/DeleteWallet/index.tsx [hunk 2]
```diff
@@ -138,7 +135,7 @@ const DeleteWallet = memo(() => {
                 </div>
                 <div className={classes.info}>
                     <div className={classes.iconContainer}>
-                        <WalletIcon networkIcon={networkDescriptor?.icon} providerIcon={providerDescriptor?.icon} />
+                        <WalletIcon />
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Wallet/DeleteWallet/index.tsx:138]: This will render nothing.

Comment (guanbinrui) [reply packages/mask/src/extension/popups/pages/Wallet/DeleteWallet/index.tsx:138]: Pass it!

**Pull request merged**